Nagaza/Sources/Presentation/Feature/Login/View/LoginViewController.swift

```diff
@@ -45,27 +145,52 @@ final class LoginViewController: UIViewController, Alertable {
     }
     
     private func setupView() {
-        self.view.backgroundColor = .white
+        view.backgroundColor = .white
+        view.addSubview(temporaryMoveHomeBtn)
+        view.addSubview(appLogo)
+        view.addSubview(snsLoginStackView)
```

Comment:
UIView Extension에 있는 함수로 적으면 더 좋을 것 같아요!
```suggestion
        view.addSubviews([temporaryMoveHomeBtn, appLogo, snsLoginStackView])
```